TextField/TextField.jsx

```diff
@@ -199,6 +205,13 @@ class TextField extends Component {
       props.id = 'tf-' + this.id;
     }
 
+    const intermediateStyle = Object.assign(
```

Comment:
why this?

Reply:
Else when scaling up the intermediate layer (the actual input, when using helper text) won't scale up, when scaling up the outer `<div>`.
```
__________________________________
|  |-----------|                 |
|  | input     |  outer div      |
|  |-----------|                 |
|  <helpertext>                  |
__________________________________
```

Reply:
That my personal use case, why I added the outerStyle